### DISCUSSION TITLE
Needs to update examples for Storage in Wiki

### DISCUSSION THREAD

**Discussion by syntaxerror247** [GodotNuts/GodotFirebase#427] [General]
Examples in Wiki are outdated for Storage API. After recent updates, functions are now coroutine so it must be called with await.

For eg.
```
var upload_task = Firebase.Storage.ref("Firebasetester/upload/icon.png").put_file("res://icon.png")
await upload_task.task_finished
```
Should be updated to:
```
var upload_task = await Firebase.Storage.ref("Firebasetester/upload/icon.png").put_file("res://icon.png")
```

**Comment by WolfgangSenff** [comment]
Ahh crap, you're right. I totally forgot to update the wiki for that. My bad. I will try to do so today if I can find the time. Good find!

**Comment by WolfgangSenff** [comment]
Okay, that is now up-to-date, so I'm closing this. See https://github.com/GodotNuts/GodotFirebase/wiki/Storage-2.0 for how to use storage in 4.x. Also, if you wouldn't mind, come onto the Discord server if you're not there already and let me know if you'd like to see this separation for Firestore as well. I didn't really have time in the past to go through and separate everything out, but I think I could now and it feels better to me, even if I end up going back and implementing the 4.x pattern in 3.x like I did with Firestore (relevant because it would mean essentially rewriting the entire 3.x page for Storage again).
